packages/flutter/test/widgets/scrollbar_test.dart

```diff
@@ -955,4 +955,33 @@ void main() {
         ),
     );
   });
+
+  testWidgets('RawScrollbar.isAlwaysShown asserts that a ScrollPosition is attached', (WidgetTester tester) async {
```

Comment:
This test doesn't seem to use isAlwaysShown, unless I'm misunderstanding.

Reply:
Ah! 😱  Thank you!